packages/manager/modules/bm-server-components/src/consumption-tile/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import '@ovh-ux/ui-kit';
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+import ovhManagerCore from '@ovh-ux/manager-core';
+
+import component from './component';
+
+const moduleName = 'ovhManagerBmServerComponentsConsumptionTile';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'pascalprecht.translate',
+    'ngTranslateAsyncLoader',
```

Comment:
Do you really need `ngTranslateAsyncLoader` module here ?

Reply:
I need to check, because i done the copy / past but maybe it isn't required

Reply:
I removed it, it's no required.

Reply:
Did you forget to remove it ?

Reply:
Now is good, in fact i removed the import but i forget this.